guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -1,32 +1,27 @@
 [id="Installing_the_OpenSCAP_Plugin_{context}"]
 = Installing the OpenSCAP Plug-in
 
-You can install and enable the OpenSCAP plug-in to generate OpenSCAP compliance reports.
-The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP smart proxy plug-in, and the OpenSCAP Hammer CLI plug-in.
+You can install and enable the OpenSCAP plug-in to generate SCAP compliance reports.
+The OpenSCAP plug-in consists of the OpenSCAP {Project} plug-in, the OpenSCAP Hammer CLI plug-in, the OpenSCAP {SmartProxy} plug-in, and modules for policy deployment.
 
 .Procedure
 . Install the OpenSCAP plug-in on your {ProjectServer}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-openscap --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-puppet-module true
```

Comment:
I could not run this on Foreman 3.5:

````
# foreman-installer --tuning development --enable-foreman-plugin-openscap --enable-foreman-cli-openscap --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-ansible-module true --foreman-proxy-plugin-openscap-puppet-module true
2023-09-29 08:32:37 [NOTICE] [root] Loading installer configuration. This will take some time.
2023-09-29 08:32:42 [NOTICE] [root] Running installer with log based terminal output at level NOTICE.
2023-09-29 08:32:42 [NOTICE] [root] Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG. See --full-help for definitions.
ERROR: Unrecognised option '--foreman-proxy-plugin-openscap-ansible-module'
````

Reply:
Thank you so much for the testing!
This is the correct behaviour, because the ansible argument was introduced in 3.6.
I forgot to remove it in this doc module, my bad! Will fix that.

Reply:
@maximiliankolb The command should work now.

Reply:
works!